resources/dicts/patrols/general/border.json

```
@@ -2317,7 +2317,7 @@
     "biome": ["any"],
     "season": ["any"],
     "types": ["border"],
-    "tags": ["new_cat"],
+    "tags": [],
```

```suggestion
    "tags": ["new_cat],
```
this tag needs to stay!